fe/fe-core/src/main/java/org/apache/doris/catalog/TabletInvertedIndex.java

```diff
@@ -335,48 +337,81 @@ private boolean needSync(Replica replicaInFe, TTabletInfo backendTabletInfo) {
         return false;
     }
 
-    private boolean needChangeCooldownConf(TabletMeta tabletMeta, TTabletInfo beTabletInfo) {
-        if (!beTabletInfo.isIsCooldown()) {
-            return false;
-        }
-        // check cooldown type in fe and be, they need to be the same.
-        if (tabletMeta.getCooldownReplicaId() != beTabletInfo.getCooldownReplicaId()) {
-            LOG.warn("cooldownReplicaId is wrong for tablet: {}, Fe: {}, Be: {}", beTabletInfo.getTabletId(),
-                    tabletMeta.getCooldownReplicaId(), beTabletInfo.getCooldownReplicaId());
-            return true;
+    private void handleCooldownConf(TabletMeta tabletMeta, TTabletInfo beTabletInfo,
+            List<CooldownConf> cooldownConfToPush, List<CooldownConf> cooldownConfToUpdate) {
+        if (!beTabletInfo.isSetCooldownReplicaId()) {
+            return;
         }
-        // check cooldown type in one tablet, One UPLOAD_DATA is needed in the replicas.
-        long stamp = readLock();
+        Tablet tablet;
         try {
-            boolean replicaInvalid = true;
-            Map<Long, Replica> replicaMap = replicaMetaTable.row(beTabletInfo.getTabletId());
-            for (Map.Entry<Long, Replica> entry : replicaMap.entrySet()) {
-                if (entry.getValue().getId() == beTabletInfo.getCooldownReplicaId()) {
-                    replicaInvalid = false;
-                    break;
-                }
+            OlapTable table = (OlapTable) Env.getCurrentInternalCatalog().getDbNullable(tabletMeta.getDbId())
+                    .getTable(tabletMeta.getTableId())
+                    .get();
+            table.readLock();
+            try {
+                tablet = table.getPartition(tabletMeta.getPartitionId()).getIndex(tabletMeta.getIndexId())
+                        .getTablet(beTabletInfo.tablet_id);
+            } finally {
+                table.readUnlock();
             }
-            if (replicaInvalid) {
-                return true;
+        } catch (RuntimeException e) {
+            LOG.warn("failed to get tablet. tabletId={}", beTabletInfo.tablet_id);
+            return;
+        }
+        Pair<Long, Long> cooldownConf = tablet.getCooldownConf();
```

Comment:
Create a class named CooldownConf is better, it will be more readability.
And it is easier to add new element to CooldownConf.